draco/asp/soft.lp

```diff
@@ -334,33 +334,80 @@ preference(c_c_text,M) :-
 % @soft(c_d_point) Continuous by discrete for point mark.
 preference(c_d_point,M) :-
     attribute(is_c_d,M),
+    not attribute(no_overlap,M),
```

Comment:
it should not be the case that there is not no overlap = it should not be the case that there is overlap. Can we use a different helper to make this simpler?

Reply:
What's the point of adding the overlap fact into this preference?

Reply:
I think `c_d_point` and `c_d_point_no_overlap` need to be mutually excluded. Otherwise `c_d_point_no_overlap` will be penalized twice for both `c_d_point` and `c_d_point_no_overlap`?

Reply:
It doesn't have to be. If we have two weights for mutually exclusive preferences an and b, then we can always simulate that with with a negative weight. It may make sense to make constraints mutually exclusive and we should re-evaluate as we improve the weights.